R/utils.R

```diff
@@ -109,8 +109,9 @@ create_stressdata_masterdata_file_paths <- function(data_prep_timestamp, twodii_
   path_parent <- path_dropbox_2dii("PortCheck", "00_Data", "07_AnalysisInputs", data_prep_timestamp)
 
   paths <- list(
-    "stress_test_masterdata_debt.rda",
-    "stress_test_masterdata_ownership.rda"
+    "stress_test_masterdata_debt.rds",
+    "stress_test_masterdata_ownership.rds",
+    "stress_test_masterdata_credit.rds"
```

Comment:
Just wondering, the now available data are not yet to actually be read in when running the loanbook flow?

Reply:
no, but definitely will be the next change required on the loan book side

Reply:
https://dev.azure.com/2DegreesInvesting/2DegreesInvesting/_workitems/edit/879